x/pool-incentives/keeper/keeper.go

```diff
@@ -100,29 +101,26 @@ func (k Keeper) CreateConcentratedLiquidityPoolGauge(ctx sdk.Context, poolId uin
 		return fmt.Errorf("pool %d is not concentrated liquidity pool", poolId)
 	}
 
-	gaugeIdCL, err := k.incentivesKeeper.CreateGauge(
+	incentivesEpoch := k.incentivesKeeper.GetEpochInfo(ctx)
+
+	_, err = k.incentivesKeeper.CreateGauge(
 		ctx,
 		true,
 		k.accountKeeper.GetModuleAddress(types.ModuleName),
 		sdk.Coins{},
-		// dummy variable so that the existing logic does not break
-		// CreateGauge checks if LockQueryType is `ByDuration` or not, we bypass this check by passing
-		// lockQueryType as byTime. Although we do not need this check, we still cannot pass empty struct.
 		lockuptypes.QueryCondition{
-			LockQueryType: lockuptypes.ByTime,
+			LockQueryType: lockuptypes.NoLock,
 			Denom:         appParams.BaseCoinUnit,
```

Comment:
i dont think this is necessary as per this discussion https://github.com/osmosis-labs/osmosis/pull/5459/files#r1222354794
```suggestion
```